Remove OLB_ROOT definition from example makefiles. The env variable should be used.

Author: Loris Ercole <loris.ercole@epfl.ch>

diff --git a/examples/adsorption/adsorption3d/Makefile b/examples/adsorption/adsorption3d/Makefile
index 55d1af8..9300472 100644
--- a/examples/adsorption/adsorption3d/Makefile
+++ b/examples/adsorption/adsorption3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = adsorption3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/adsorption/microMixer3d/Makefile b/examples/adsorption/microMixer3d/Makefile
index 925e3ed..cf30444 100644
--- a/examples/adsorption/microMixer3d/Makefile
+++ b/examples/adsorption/microMixer3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = microMixer3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/advectionDiffusionReaction/advectionDiffusion1d/Makefile b/examples/advectionDiffusionReaction/advectionDiffusion1d/Makefile
index d2d9ac9..8509f69 100644
--- a/examples/advectionDiffusionReaction/advectionDiffusion1d/Makefile
+++ b/examples/advectionDiffusionReaction/advectionDiffusion1d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = advectionDiffusion1d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/advectionDiffusionReaction/advectionDiffusion2d/Makefile b/examples/advectionDiffusionReaction/advectionDiffusion2d/Makefile
index 855c803..cf818c1 100644
--- a/examples/advectionDiffusionReaction/advectionDiffusion2d/Makefile
+++ b/examples/advectionDiffusionReaction/advectionDiffusion2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = advectionDiffusion2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/advectionDiffusionReaction/advectionDiffusion3d/Makefile b/examples/advectionDiffusionReaction/advectionDiffusion3d/Makefile
index f4de046..c0c3da5 100644
--- a/examples/advectionDiffusionReaction/advectionDiffusion3d/Makefile
+++ b/examples/advectionDiffusionReaction/advectionDiffusion3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = advectionDiffusion3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/advectionDiffusionReaction/convectedPlate3d/Makefile b/examples/advectionDiffusionReaction/convectedPlate3d/Makefile
index 9e6e08a..e6de7df 100644
--- a/examples/advectionDiffusionReaction/convectedPlate3d/Makefile
+++ b/examples/advectionDiffusionReaction/convectedPlate3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = convectedPlate3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/advectionDiffusionReaction/laminarReactiveTmixer/Makefile b/examples/advectionDiffusionReaction/laminarReactiveTmixer/Makefile
index 9dac24e..33b59e6 100644
--- a/examples/advectionDiffusionReaction/laminarReactiveTmixer/Makefile
+++ b/examples/advectionDiffusionReaction/laminarReactiveTmixer/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = laminarReactiveTmixer
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/advectionDiffusionReaction/longitudinalMixing3d/Makefile b/examples/advectionDiffusionReaction/longitudinalMixing3d/Makefile
index 2df9530..97610ff 100644
--- a/examples/advectionDiffusionReaction/longitudinalMixing3d/Makefile
+++ b/examples/advectionDiffusionReaction/longitudinalMixing3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = longitudinalMixing3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/electroChemistry/electroosmosis/Makefile b/examples/electroChemistry/electroosmosis/Makefile
index a0c144f..3f27189 100644
--- a/examples/electroChemistry/electroosmosis/Makefile
+++ b/examples/electroChemistry/electroosmosis/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = electroosmosis
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/electroChemistry/poissonNernstPlanck/Makefile b/examples/electroChemistry/poissonNernstPlanck/Makefile
index 51e3049..2ecf67d 100644
--- a/examples/electroChemistry/poissonNernstPlanck/Makefile
+++ b/examples/electroChemistry/poissonNernstPlanck/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = poissonNernstPlanck
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/forBeginners/cavity2d/Makefile b/examples/forBeginners/cavity2d/Makefile
index 5e03620..fe92edb 100644
--- a/examples/forBeginners/cavity2d/Makefile
+++ b/examples/forBeginners/cavity2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cavity2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/forBeginners/cylinder2d/Makefile b/examples/forBeginners/cylinder2d/Makefile
index 35a9dc4..b59d97a 100644
--- a/examples/forBeginners/cylinder2d/Makefile
+++ b/examples/forBeginners/cylinder2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cylinder2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/forBeginners/minimal3d/Makefile b/examples/forBeginners/minimal3d/Makefile
index 46cf093..b0096b2 100644
--- a/examples/forBeginners/minimal3d/Makefile
+++ b/examples/forBeginners/minimal3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = minimal3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/forBeginners/pipeWithValve3d/Makefile b/examples/forBeginners/pipeWithValve3d/Makefile
index 35d45b6..92b54ea 100644
--- a/examples/forBeginners/pipeWithValve3d/Makefile
+++ b/examples/forBeginners/pipeWithValve3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = pipeWithValve3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/freeSurface/basicRisingBubble3d/Makefile b/examples/freeSurface/basicRisingBubble3d/Makefile
index 62ddeed..386b160 100644
--- a/examples/freeSurface/basicRisingBubble3d/Makefile
+++ b/examples/freeSurface/basicRisingBubble3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = basicRisingBubble3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/freeSurface/breakingDam2d/Makefile b/examples/freeSurface/breakingDam2d/Makefile
index 79734b8..5ca71a7 100644
--- a/examples/freeSurface/breakingDam2d/Makefile
+++ b/examples/freeSurface/breakingDam2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = breakingDam2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/freeSurface/breakingDam3d/Makefile b/examples/freeSurface/breakingDam3d/Makefile
index f3d5e25..085007a 100644
--- a/examples/freeSurface/breakingDam3d/Makefile
+++ b/examples/freeSurface/breakingDam3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = breakingDam3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/freeSurface/deepFallingDrop2d/Makefile b/examples/freeSurface/deepFallingDrop2d/Makefile
index d99acb7..e7db63e 100644
--- a/examples/freeSurface/deepFallingDrop2d/Makefile
+++ b/examples/freeSurface/deepFallingDrop2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = deepFallingDrop
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/freeSurface/fallingDrop2d/Makefile b/examples/freeSurface/fallingDrop2d/Makefile
index e8bb15a..c05bc3d 100644
--- a/examples/freeSurface/fallingDrop2d/Makefile
+++ b/examples/freeSurface/fallingDrop2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = fallingDrop2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/freeSurface/fallingDrop3d/Makefile b/examples/freeSurface/fallingDrop3d/Makefile
index 8f2ea66..0a8f9a3 100644
--- a/examples/freeSurface/fallingDrop3d/Makefile
+++ b/examples/freeSurface/fallingDrop3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = fallingDrop3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/freeSurface/rayleighInstability3d/Makefile b/examples/freeSurface/rayleighInstability3d/Makefile
index 4bd58be..de8d576 100644
--- a/examples/freeSurface/rayleighInstability3d/Makefile
+++ b/examples/freeSurface/rayleighInstability3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = rayleighInstability
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/fsi/deformableParticleInShearFlow3d/Makefile b/examples/fsi/deformableParticleInShearFlow3d/Makefile
index 6a11e4c..3b83cec 100644
--- a/examples/fsi/deformableParticleInShearFlow3d/Makefile
+++ b/examples/fsi/deformableParticleInShearFlow3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = deformableParticleInShearFlow3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/fsi/rigidValve2d/Makefile b/examples/fsi/rigidValve2d/Makefile
index 9e22247..715ddff 100644
--- a/examples/fsi/rigidValve2d/Makefile
+++ b/examples/fsi/rigidValve2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = rigidValve2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/fsi/taylorCouette3d/Makefile b/examples/fsi/taylorCouette3d/Makefile
index 61116ef..db1f2a0 100644
--- a/examples/fsi/taylorCouette3d/Makefile
+++ b/examples/fsi/taylorCouette3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = taylorCouette3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/gridRefinement/cylinder2d/Makefile b/examples/gridRefinement/cylinder2d/Makefile
index 35a9dc4..b59d97a 100644
--- a/examples/gridRefinement/cylinder2d/Makefile
+++ b/examples/gridRefinement/cylinder2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cylinder2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/gridRefinement/sphere3d/Makefile b/examples/gridRefinement/sphere3d/Makefile
index 24dcda8..d65332e 100644
--- a/examples/gridRefinement/sphere3d/Makefile
+++ b/examples/gridRefinement/sphere3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = sphere3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/bstep2d/Makefile b/examples/laminar/bstep2d/Makefile
index 3bba591..7f46fc2 100644
--- a/examples/laminar/bstep2d/Makefile
+++ b/examples/laminar/bstep2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = bstep2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/bstep3d/Makefile b/examples/laminar/bstep3d/Makefile
index 0ce82f2..8882f5c 100644
--- a/examples/laminar/bstep3d/Makefile
+++ b/examples/laminar/bstep3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = bstep3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/cavity2d/Makefile b/examples/laminar/cavity2d/Makefile
index 5e03620..fe92edb 100644
--- a/examples/laminar/cavity2d/Makefile
+++ b/examples/laminar/cavity2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cavity2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/cavity2dSolver/Makefile b/examples/laminar/cavity2dSolver/Makefile
index 5e03620..fe92edb 100644
--- a/examples/laminar/cavity2dSolver/Makefile
+++ b/examples/laminar/cavity2dSolver/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cavity2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/cavity3d/Makefile b/examples/laminar/cavity3d/Makefile
index 8b4e934..95c9314 100644
--- a/examples/laminar/cavity3d/Makefile
+++ b/examples/laminar/cavity3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cavity3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/cavity3dBenchmark/Makefile b/examples/laminar/cavity3dBenchmark/Makefile
index 8b4e934..95c9314 100644
--- a/examples/laminar/cavity3dBenchmark/Makefile
+++ b/examples/laminar/cavity3dBenchmark/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cavity3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/cylinder2d/Makefile b/examples/laminar/cylinder2d/Makefile
index 35a9dc4..b59d97a 100644
--- a/examples/laminar/cylinder2d/Makefile
+++ b/examples/laminar/cylinder2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cylinder2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/cylinder3d/Makefile b/examples/laminar/cylinder3d/Makefile
index 48959d3..8563ee0 100644
--- a/examples/laminar/cylinder3d/Makefile
+++ b/examples/laminar/cylinder3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cylinder3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/nonNewtonianPoiseuille3d/Makefile b/examples/laminar/nonNewtonianPoiseuille3d/Makefile
index c5b2be8..0e15d74 100644
--- a/examples/laminar/nonNewtonianPoiseuille3d/Makefile
+++ b/examples/laminar/nonNewtonianPoiseuille3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = nonNewtonianPoiseuille3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/poiseuille2d/Makefile b/examples/laminar/poiseuille2d/Makefile
index 5a8414f..33363df 100644
--- a/examples/laminar/poiseuille2d/Makefile
+++ b/examples/laminar/poiseuille2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = poiseuille2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/poiseuille3d/Makefile b/examples/laminar/poiseuille3d/Makefile
index b45ca87..0fad9d0 100644
--- a/examples/laminar/poiseuille3d/Makefile
+++ b/examples/laminar/poiseuille3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = poiseuille3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/laminar/powerLaw2d/Makefile b/examples/laminar/powerLaw2d/Makefile
index 228bb15..e8577cf 100644
--- a/examples/laminar/powerLaw2d/Makefile
+++ b/examples/laminar/powerLaw2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = powerLaw2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/laminar/testFlow3dSolver/Makefile b/examples/laminar/testFlow3dSolver/Makefile
index 3039a3d..cf6af3b 100644
--- a/examples/laminar/testFlow3dSolver/Makefile
+++ b/examples/laminar/testFlow3dSolver/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = testFlow3dSolver
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/microfluidics/knudsenPump3d/Makefile b/examples/microfluidics/knudsenPump3d/Makefile
index 9f9427f..7dd24c6 100644
--- a/examples/microfluidics/knudsenPump3d/Makefile
+++ b/examples/microfluidics/knudsenPump3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = knudsenPump3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/microfluidics/microChannel2d/Makefile b/examples/microfluidics/microChannel2d/Makefile
index 4dd8887..168fdba 100644
--- a/examples/microfluidics/microChannel2d/Makefile
+++ b/examples/microfluidics/microChannel2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = microChannel
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/microfluidics/microChannel3d/Makefile b/examples/microfluidics/microChannel3d/Makefile
index 4e67ada..bbd8b06 100644
--- a/examples/microfluidics/microChannel3d/Makefile
+++ b/examples/microfluidics/microChannel3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = microChannel3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/airBubbleCoalescence3d/Makefile b/examples/multiComponent/airBubbleCoalescence3d/Makefile
index 050df02..b569d1c 100644
--- a/examples/multiComponent/airBubbleCoalescence3d/Makefile
+++ b/examples/multiComponent/airBubbleCoalescence3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = airBubbleCoalescence3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/binaryShearFlow2d/Makefile b/examples/multiComponent/binaryShearFlow2d/Makefile
index e0afccb..f2bd0c1 100644
--- a/examples/multiComponent/binaryShearFlow2d/Makefile
+++ b/examples/multiComponent/binaryShearFlow2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = binaryShearFlow2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/bubbleChannel2d/Makefile b/examples/multiComponent/bubbleChannel2d/Makefile
index 495c957..9140b90 100644
--- a/examples/multiComponent/bubbleChannel2d/Makefile
+++ b/examples/multiComponent/bubbleChannel2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = bubbleChannel2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/multiComponent/contactAngle2d/Makefile b/examples/multiComponent/contactAngle2d/Makefile
index bd7d631..fbf0bb6 100644
--- a/examples/multiComponent/contactAngle2d/Makefile
+++ b/examples/multiComponent/contactAngle2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = contactAngle2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/contactAngle3d/Makefile b/examples/multiComponent/contactAngle3d/Makefile
index cf2ff85..2d11b44 100644
--- a/examples/multiComponent/contactAngle3d/Makefile
+++ b/examples/multiComponent/contactAngle3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = contactAngle3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/multiComponent/dropletSplashing2d/Makefile b/examples/multiComponent/dropletSplashing2d/Makefile
index 65ee76e..05a4507 100644
--- a/examples/multiComponent/dropletSplashing2d/Makefile
+++ b/examples/multiComponent/dropletSplashing2d/Makefile
@@ -1,4 +1,3 @@
 EXAMPLE = dropletSplashing2d
-OLB_ROOT := ../../..
 OLB_MIXED_MODE_INCLUDE_CPP := ON
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/dropletSplashing3d/Makefile b/examples/multiComponent/dropletSplashing3d/Makefile
index d223cb0..1116173 100644
--- a/examples/multiComponent/dropletSplashing3d/Makefile
+++ b/examples/multiComponent/dropletSplashing3d/Makefile
@@ -1,4 +1,3 @@
 EXAMPLE = dropletSplashing3d
-OLB_ROOT := ../../..
 OLB_MIXED_MODE_INCLUDE_CPP := ON
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/flatInterfaceEOC2d/Makefile b/examples/multiComponent/flatInterfaceEOC2d/Makefile
index 0b6e719..242cb84 100644
--- a/examples/multiComponent/flatInterfaceEOC2d/Makefile
+++ b/examples/multiComponent/flatInterfaceEOC2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = flatInterfaceEOC2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/fourRollMill2d/Makefile b/examples/multiComponent/fourRollMill2d/Makefile
index 863bf2d..4356393 100644
--- a/examples/multiComponent/fourRollMill2d/Makefile
+++ b/examples/multiComponent/fourRollMill2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = fourRollMill2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/isothermalEvaporation2d/Makefile b/examples/multiComponent/isothermalEvaporation2d/Makefile
index 2a36c16..9fe47ce 100644
--- a/examples/multiComponent/isothermalEvaporation2d/Makefile
+++ b/examples/multiComponent/isothermalEvaporation2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = isothermalEvaporation2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/microFluidics2d/Makefile b/examples/multiComponent/microFluidics2d/Makefile
index 12c401f..38b9d65 100644
--- a/examples/multiComponent/microFluidics2d/Makefile
+++ b/examples/multiComponent/microFluidics2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = microFluidics2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/multiComponent/phaseSeparation2d/Makefile b/examples/multiComponent/phaseSeparation2d/Makefile
index b1c2934..2f50146 100644
--- a/examples/multiComponent/phaseSeparation2d/Makefile
+++ b/examples/multiComponent/phaseSeparation2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = phaseSeparation2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/multiComponent/phaseSeparation3d/Makefile b/examples/multiComponent/phaseSeparation3d/Makefile
index 2ad9cf4..e77064c 100644
--- a/examples/multiComponent/phaseSeparation3d/Makefile
+++ b/examples/multiComponent/phaseSeparation3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = phaseSeparation3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/multiComponent/rayleighTaylor2d/Makefile b/examples/multiComponent/rayleighTaylor2d/Makefile
index 5f5766b..1076155 100644
--- a/examples/multiComponent/rayleighTaylor2d/Makefile
+++ b/examples/multiComponent/rayleighTaylor2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = rayleighTaylor2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/multiComponent/rayleighTaylor3d/Makefile b/examples/multiComponent/rayleighTaylor3d/Makefile
index 86dcba5..a2803a7 100644
--- a/examples/multiComponent/rayleighTaylor3d/Makefile
+++ b/examples/multiComponent/rayleighTaylor3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = rayleighTaylor3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/multiComponent/waterAirFlatInterface2d/Makefile b/examples/multiComponent/waterAirFlatInterface2d/Makefile
index 936da48..a056561 100644
--- a/examples/multiComponent/waterAirFlatInterface2d/Makefile
+++ b/examples/multiComponent/waterAirFlatInterface2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = waterAirFlatInterface2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/multiComponent/youngLaplace2d/Makefile b/examples/multiComponent/youngLaplace2d/Makefile
index 36a3c52..69ee1d4 100644
--- a/examples/multiComponent/youngLaplace2d/Makefile
+++ b/examples/multiComponent/youngLaplace2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = youngLaplace2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/multiComponent/youngLaplace3d/Makefile b/examples/multiComponent/youngLaplace3d/Makefile
index 5331d4c..752ceb0 100644
--- a/examples/multiComponent/youngLaplace3d/Makefile
+++ b/examples/multiComponent/youngLaplace3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = youngLaplace3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/noise/gausspulse3d/Makefile b/examples/noise/gausspulse3d/Makefile
index eeeb0b3..bf20490 100644
--- a/examples/noise/gausspulse3d/Makefile
+++ b/examples/noise/gausspulse3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = gausspulse3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/optimization/domainId3d/Makefile b/examples/optimization/domainId3d/Makefile
index f8d191c..b7409ff 100644
--- a/examples/optimization/domainId3d/Makefile
+++ b/examples/optimization/domainId3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = domainId3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/optimization/domainId3dSolver/Makefile b/examples/optimization/domainId3dSolver/Makefile
index 457fab3..f5594bf 100644
--- a/examples/optimization/domainId3dSolver/Makefile
+++ b/examples/optimization/domainId3dSolver/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = domainIdentification3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/optimization/parameterIdentificationPoiseuille2d/Makefile b/examples/optimization/parameterIdentificationPoiseuille2d/Makefile
index 8b55e9f..f6edd3b 100644
--- a/examples/optimization/parameterIdentificationPoiseuille2d/Makefile
+++ b/examples/optimization/parameterIdentificationPoiseuille2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = parameterIdentificationPoiseuille2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/optimization/pipeBendSolver2d/Makefile b/examples/optimization/pipeBendSolver2d/Makefile
index 4d24f06..6031a56 100644
--- a/examples/optimization/pipeBendSolver2d/Makefile
+++ b/examples/optimization/pipeBendSolver2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = pipeBend2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/optimization/showcaseADf/Makefile b/examples/optimization/showcaseADf/Makefile
index be752d6..3eccf25 100644
--- a/examples/optimization/showcaseADf/Makefile
+++ b/examples/optimization/showcaseADf/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = showcaseADf
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/optimization/showcaseRosenbrock/Makefile b/examples/optimization/showcaseRosenbrock/Makefile
index 559953e..83422a3 100644
--- a/examples/optimization/showcaseRosenbrock/Makefile
+++ b/examples/optimization/showcaseRosenbrock/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = showcaseRosenbrock
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/optimization/testFlowOpti3d/Makefile b/examples/optimization/testFlowOpti3d/Makefile
index c4d18b4..5f99fad 100644
--- a/examples/optimization/testFlowOpti3d/Makefile
+++ b/examples/optimization/testFlowOpti3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = testFlowOpti3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/optimization/testFlowOpti3dSolver/Makefile b/examples/optimization/testFlowOpti3dSolver/Makefile
index c4d18b4..5f99fad 100644
--- a/examples/optimization/testFlowOpti3dSolver/Makefile
+++ b/examples/optimization/testFlowOpti3dSolver/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = testFlowOpti3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/particles/bifurcation3d/eulerEuler/Makefile b/examples/particles/bifurcation3d/eulerEuler/Makefile
index 250c21d..7acb575 100644
--- a/examples/particles/bifurcation3d/eulerEuler/Makefile
+++ b/examples/particles/bifurcation3d/eulerEuler/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = bifurcation3d
-OLB_ROOT := ../../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/particles/bifurcation3d/eulerLagrange/Makefile b/examples/particles/bifurcation3d/eulerLagrange/Makefile
index 250c21d..7acb575 100644
--- a/examples/particles/bifurcation3d/eulerLagrange/Makefile
+++ b/examples/particles/bifurcation3d/eulerLagrange/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = bifurcation3d
-OLB_ROOT := ../../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/particles/bifurcation3d/eulerLagrange/nonSphericals/Makefile b/examples/particles/bifurcation3d/eulerLagrange/nonSphericals/Makefile
index 0fd00aa..a12e447 100644
--- a/examples/particles/bifurcation3d/eulerLagrange/nonSphericals/Makefile
+++ b/examples/particles/bifurcation3d/eulerLagrange/nonSphericals/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = bifurcation3d_non_sph
-OLB_ROOT := ../../../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/particles/dkt2d/Makefile b/examples/particles/dkt2d/Makefile
index 749eaf9..71c1489 100644
--- a/examples/particles/dkt2d/Makefile
+++ b/examples/particles/dkt2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = dkt2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/particles/settlingCube3d/Makefile b/examples/particles/settlingCube3d/Makefile
index 4a930f5..3bd44ed 100644
--- a/examples/particles/settlingCube3d/Makefile
+++ b/examples/particles/settlingCube3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = settlingCube3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/pdeSolverEoc/cylinder2dEoc/Makefile b/examples/pdeSolverEoc/cylinder2dEoc/Makefile
index c70ec12..0f2eb7d 100644
--- a/examples/pdeSolverEoc/cylinder2dEoc/Makefile
+++ b/examples/pdeSolverEoc/cylinder2dEoc/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cylinder2dEoc
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/pdeSolverEoc/cylinder3dEoc/Makefile b/examples/pdeSolverEoc/cylinder3dEoc/Makefile
index b2c3b00..0d61bca 100644
--- a/examples/pdeSolverEoc/cylinder3dEoc/Makefile
+++ b/examples/pdeSolverEoc/cylinder3dEoc/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cylinder3dEoc
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/pdeSolverEoc/poiseuille2dEoc/Makefile b/examples/pdeSolverEoc/poiseuille2dEoc/Makefile
index 73d273f..9fb182a 100644
--- a/examples/pdeSolverEoc/poiseuille2dEoc/Makefile
+++ b/examples/pdeSolverEoc/poiseuille2dEoc/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = poiseuille2dEoc
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/pdeSolverEoc/poiseuille3dEoc/Makefile b/examples/pdeSolverEoc/poiseuille3dEoc/Makefile
index 63b3876..7e9b583 100644
--- a/examples/pdeSolverEoc/poiseuille3dEoc/Makefile
+++ b/examples/pdeSolverEoc/poiseuille3dEoc/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = poiseuille3dEoc
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/pdeSolverEoc/poisson/Makefile b/examples/pdeSolverEoc/poisson/Makefile
index cf9a3d1..6d4c752 100644
--- a/examples/pdeSolverEoc/poisson/Makefile
+++ b/examples/pdeSolverEoc/poisson/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = poisson1d
-OLB_ROOT := ../../../
 include $(OLB_ROOT)/default.mk
diff --git a/examples/porousMedia/gasStorage2d/Makefile b/examples/porousMedia/gasStorage2d/Makefile
index 253b62f..7e0d4a2 100644
--- a/examples/porousMedia/gasStorage2d/Makefile
+++ b/examples/porousMedia/gasStorage2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = gasStorage2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/porousMedia/porousPoiseuille2d/Makefile b/examples/porousMedia/porousPoiseuille2d/Makefile
index a2c007c..034e94f 100644
--- a/examples/porousMedia/porousPoiseuille2d/Makefile
+++ b/examples/porousMedia/porousPoiseuille2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = porousPoiseuille2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/porousMedia/porousPoiseuille3d/Makefile b/examples/porousMedia/porousPoiseuille3d/Makefile
index a44dddd..32bffaf 100644
--- a/examples/porousMedia/porousPoiseuille3d/Makefile
+++ b/examples/porousMedia/porousPoiseuille3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = porousPoiseuille3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/porousMedia/resolvedRock3d/Makefile b/examples/porousMedia/resolvedRock3d/Makefile
index d7e18dc..be74937 100644
--- a/examples/porousMedia/resolvedRock3d/Makefile
+++ b/examples/porousMedia/resolvedRock3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = resolvedRock3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/radiativeTransport/sphere3d/Makefile b/examples/radiativeTransport/sphere3d/Makefile
index 24dcda8..d65332e 100644
--- a/examples/radiativeTransport/sphere3d/Makefile
+++ b/examples/radiativeTransport/sphere3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = sphere3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/showCase/centrifugalPump3d/Makefile b/examples/showCase/centrifugalPump3d/Makefile
index 4750186..6aaa700 100644
--- a/examples/showCase/centrifugalPump3d/Makefile
+++ b/examples/showCase/centrifugalPump3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = pump3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/showCase/urbanDigitalTwin3d/Makefile b/examples/showCase/urbanDigitalTwin3d/Makefile
index 6ff4888..0064241 100644
--- a/examples/showCase/urbanDigitalTwin3d/Makefile
+++ b/examples/showCase/urbanDigitalTwin3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = urbanDigitalTwin
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/thermal/galliumMelting2d/Makefile b/examples/thermal/galliumMelting2d/Makefile
index 06687ab..2a8451b 100644
--- a/examples/thermal/galliumMelting2d/Makefile
+++ b/examples/thermal/galliumMelting2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = galliumMelting2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/thermal/porousPlate2d/Makefile b/examples/thermal/porousPlate2d/Makefile
index 986c0c4..856f03c 100644
--- a/examples/thermal/porousPlate2d/Makefile
+++ b/examples/thermal/porousPlate2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = porousPlate2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/thermal/porousPlate3d/Makefile b/examples/thermal/porousPlate3d/Makefile
index 44414b6..c7f0c44 100644
--- a/examples/thermal/porousPlate3d/Makefile
+++ b/examples/thermal/porousPlate3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = porousPlate3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/thermal/porousPlate3dSolver/Makefile b/examples/thermal/porousPlate3dSolver/Makefile
index e6b0ab1..2994e6d 100644
--- a/examples/thermal/porousPlate3dSolver/Makefile
+++ b/examples/thermal/porousPlate3dSolver/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = porousPlate3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/thermal/rayleighBenard2d/Makefile b/examples/thermal/rayleighBenard2d/Makefile
index f91493b..596ce12 100644
--- a/examples/thermal/rayleighBenard2d/Makefile
+++ b/examples/thermal/rayleighBenard2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = rayleighBenard2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/thermal/rayleighBenard3d/Makefile b/examples/thermal/rayleighBenard3d/Makefile
index 59221ea..c1e6b7d 100644
--- a/examples/thermal/rayleighBenard3d/Makefile
+++ b/examples/thermal/rayleighBenard3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = rayleighBenard3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/thermal/squareCavity2d/Makefile b/examples/thermal/squareCavity2d/Makefile
index df0de2e..0c919b6 100644
--- a/examples/thermal/squareCavity2d/Makefile
+++ b/examples/thermal/squareCavity2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = squareCavity2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/thermal/squareCavity3d/Makefile b/examples/thermal/squareCavity3d/Makefile
index bf81b9e..ecf8118 100644
--- a/examples/thermal/squareCavity3d/Makefile
+++ b/examples/thermal/squareCavity3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = squareCavity3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/thermal/stefanMelting2d/Makefile b/examples/thermal/stefanMelting2d/Makefile
index 2ab9763..3f3490d 100644
--- a/examples/thermal/stefanMelting2d/Makefile
+++ b/examples/thermal/stefanMelting2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = stefanMelting2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/turbulence/airfoil2d/Makefile b/examples/turbulence/airfoil2d/Makefile
index 687d5e8..bbc81a4 100644
--- a/examples/turbulence/airfoil2d/Makefile
+++ b/examples/turbulence/airfoil2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = airfoil2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/turbulence/aorta3d/Makefile b/examples/turbulence/aorta3d/Makefile
index 752dd9c..e65843e 100644
--- a/examples/turbulence/aorta3d/Makefile
+++ b/examples/turbulence/aorta3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = aorta3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/turbulence/channel3d/Makefile b/examples/turbulence/channel3d/Makefile
index 03d2859..001d538 100644
--- a/examples/turbulence/channel3d/Makefile
+++ b/examples/turbulence/channel3d/Makefile
@@ -1,4 +1,3 @@
 EXAMPLE = channel3d
-OLB_ROOT := ../../..
 OLB_MIXED_MODE_INCLUDE_CPP := ON
 include $(OLB_ROOT)/default.mk
diff --git a/examples/turbulence/nozzle3d/Makefile b/examples/turbulence/nozzle3d/Makefile
index f8640c5..1f6af76 100644
--- a/examples/turbulence/nozzle3d/Makefile
+++ b/examples/turbulence/nozzle3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = nozzle3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/turbulence/tgv3d/Makefile b/examples/turbulence/tgv3d/Makefile
index 4a0440d..e9036e6 100644
--- a/examples/turbulence/tgv3d/Makefile
+++ b/examples/turbulence/tgv3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = tgv3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/turbulence/venturi3d/Makefile b/examples/turbulence/venturi3d/Makefile
index 4976250..8c91d7b 100644
--- a/examples/turbulence/venturi3d/Makefile
+++ b/examples/turbulence/venturi3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = venturi3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/uncertaintyQuantification/cavity2d/Makefile b/examples/uncertaintyQuantification/cavity2d/Makefile
index 1ee3a15..65f4caa 100644
--- a/examples/uncertaintyQuantification/cavity2d/Makefile
+++ b/examples/uncertaintyQuantification/cavity2d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cavity2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
\ No newline at end of file
diff --git a/examples/uncertaintyQuantification/cavity3d/Makefile b/examples/uncertaintyQuantification/cavity3d/Makefile
index 8b4e934..95c9314 100644
--- a/examples/uncertaintyQuantification/cavity3d/Makefile
+++ b/examples/uncertaintyQuantification/cavity3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cavity3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/uncertaintyQuantification/cylinder2d/Makefile b/examples/uncertaintyQuantification/cylinder2d/Makefile
index c5af6b9..7869093 100644
--- a/examples/uncertaintyQuantification/cylinder2d/Makefile
+++ b/examples/uncertaintyQuantification/cylinder2d/Makefile
@@ -1,5 +1,4 @@
 EXAMPLE = cylinder2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
 
 # Additional flags for GSL
diff --git a/examples/uncertaintyQuantification/cylinder3d/Makefile b/examples/uncertaintyQuantification/cylinder3d/Makefile
index 48959d3..8563ee0 100644
--- a/examples/uncertaintyQuantification/cylinder3d/Makefile
+++ b/examples/uncertaintyQuantification/cylinder3d/Makefile
@@ -1,3 +1,2 @@
 EXAMPLE = cylinder3d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
diff --git a/examples/uncertaintyQuantification/tgv2d/Makefile b/examples/uncertaintyQuantification/tgv2d/Makefile
index 2295926..0c95a94 100644
--- a/examples/uncertaintyQuantification/tgv2d/Makefile
+++ b/examples/uncertaintyQuantification/tgv2d/Makefile
@@ -1,5 +1,4 @@
 EXAMPLE = tgv2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
 
 # Additional flags for GSL
diff --git a/examples/web/cylinder2d/Makefile b/examples/web/cylinder2d/Makefile
index 11c8c70..e04a21c 100644
--- a/examples/web/cylinder2d/Makefile
+++ b/examples/web/cylinder2d/Makefile
@@ -1,5 +1,4 @@
 EXAMPLE = cylinder2d
-OLB_ROOT := ../../..
 include $(OLB_ROOT)/default.mk
 
 LDFLAGS 	+= -s EXPORTED_FUNCTIONS="[_runCylinder2d,_setupCylinder2d,_getNumberCellsX,_getNumberCellsY,_getValuesCells,_getCharPhysVelocity,_getPhysMaxU,_getAvgRho,_getPhysTime,_getMLUPs,_getValuesPointer,_freeMemory,_stopTimer,_startTimer,_setOutputType,_setSimulationParameters,_main]"
